.gitignore

```diff
@@ -199,5 +199,6 @@ FakesAssemblies/
 
 .vs/
 /tools/*.dll
+/tools/LocalFeed
```

Comment:
Remove this.  LocalFeed is sometimes used, we should not ignore its contents (but they shoudl be empty except under very special circumstances